big_tests/tests/graphql_muc_SUITE.erl

```diff
@@ -287,6 +290,26 @@ ensure_muc_stopped() ->
     muc_helper:unload_muc(),
     muc_helper:unload_muc(SecondaryHostType).
 
+ensure_muc_light_started(Config) ->
+    MucLightOpts = mod_config(mod_muc_light, #{rooms_in_rosters => true,
+                                               config_schema => custom_schema()}),
+    HostType = domain_helper:host_type(),
+    dynamic_modules:ensure_modules(HostType, [{mod_muc_light, MucLightOpts}]),
+    [{muc_light_host, muc_light_helper:muc_host()} | Config].
+
+ensure_muc_light_stopped(Config) ->
+    HostType = domain_helper:host_type(),
+    dynamic_modules:ensure_modules(HostType, [{mod_muc_light, stopped}]),
+    [{muc_light_host, <<"NON_EXISTING">>} | Config].
```

Comment:
This placeholder does nothing, especially that the `Config` is ignored in `end_per_testcase`.